<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_windows_</class>
 <widget class="QMainWindow" name="_windows_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>561</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Projeto Teste</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>PUCSimual</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_remover">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
background-color: rgb(44, 44, 44);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::PenStyle::NoPen</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>106</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Aulas Previstas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Aulas Ministradas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Numero Presença</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Media</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Resultado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Falta Media</string>
       </property>
      </column>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>AI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">border-radius : 40px;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>700</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_infoSystem">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="info_arquivo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivos">
    <property name="title">
     <string>Arquivos</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvar_como"/>
   </widget>
   <widget class="QMenu" name="menuFerramentas">
    <property name="title">
     <string>Ferramentas</string>
    </property>
    <addaction name="actionOpition"/>
   </widget>
   <addaction name="menuArquivos"/>
   <addaction name="menuFerramentas"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSalvar_como">
   <property name="text">
    <string>Salvar como</string>
   </property>
  </action>
  <action name="actionExportar">
   <property name="text">
    <string>Exportar</string>
   </property>
  </action>
  <action name="actionOpition">
   <property name="text">
    <string>opções</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
